src/snippets.zig

```diff
@@ -85,6 +100,14 @@ pub const generic = [_]Snipped{
     .{ .label = "log warn", .kind = .Snippet, .text = "std.log.warn(\"$1\", .{$0});" },
     .{ .label = "log info", .kind = .Snippet, .text = "std.log.info(\"$1\", .{$0});" },
     .{ .label = "log debug", .kind = .Snippet, .text = "std.log.debug(\"$1\", .{$0});" },
+    .{ .label = "format", .kind = .Snippet, .text = 
+    \\pub fn format(
+    \\    self: ${1:@This()},
```

Comment:
Naming something `self` should be avoided if possible.
```suggestion
    \\    ${1:self}: ${2:@This()},
```